src/core/config/all_options.cpp

```diff
@@ -42,6 +42,7 @@ boost::program_options::options_description GeneralOptions() {
     po::options_description general_options("General options");
     general_options.add_options()
             (kCsvPath, po::value<std::filesystem::path>()->required(), kDCsvPath)
+            (kCsvPaths, po::value<std::vector<std::filesystem::path>>()->required(), kDCsvPaths)
```

Comment:
`->required` should be removed

Reply:
Also it seems, that we should remove `required()` for `kCsvPath`, isn't it?

Reply:
Also it seems, that we should remove `required()` for `kCsvPath`, isn't it?

Reply:
yes, that's why I highlighted both lines